helpers/html/render_html.go

```diff
@@ -4,13 +4,14 @@ import (
 	"github.com/foolin/goview/supports/ginview"
 	"github.com/gin-gonic/gin"
 	session "github.com/gutakk/go-google-scraper/helpers/session"
+	"github.com/gutakk/go-google-scraper/helpers/str"
 )
 
 func RenderWithError(c *gin.Context, status int, view string, title string, err error, data map[string]interface{}) {
 	ginview.HTML(c, status, view, gin.H{
-		"title":  title,
-		"errors": err.Error(),
-		"data":   data,
+		"title": title,
+		"error": str.CapitalizeFirst(err.Error()),
```

Comment:
Not sure if `CapitalizeFirst` is really needed. The capitalization is more of a UI concern and could be solved with CSS 😄 Sya, it was an API then it might make sense but for a web app, showing humane and well-formatted errors can be solved in the UI.

Reply:
Fixed in 2190a389763d3c2b8f161a17240f653665c92f63
I can't find any resources but I assume that Go unit test doesn't include css stuff. Thus I also need to update the unit test.

Reply:
In a unit test, the only test for lowercase, it's fine. For UI test, we also test for [case insensitivity](https://til.hashrocket.com/posts/85bda9479b-expect-a-case-insensitive-match-) to avoid tests breaking when the UI changes.